.github/workflows/k8s-content-pr-test.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name: Gate / Kubernetes Test Content Parsing
+
+on:
+  pull_request:
+    types:
+    - opened
+    - reopened
+    - synchronize
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.number || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  datastream-parsing:
+    name: XCCDF Datastream parsing
+    runs-on: ubuntu-latest
+    steps:
+      - name: Copy XCCDF files from existing content image
+        uses: nick-fields/retry@v3
```

Comment:
I think it is cleaner if [container-main](https://github.com/ComplianceAsCode/content/blob/894dfa96c30f2ac64223fb11da62fcca64bc4a4d/.github/workflows/k8s-content-pr.yaml#L43) calls this workflow when it finishes.

Alternatively, the test job could be integrated into `k8s-content-pr.yaml` and put under a conditional on successful image built/pushed.

Reply:
there is some issue with this approach, because we won't know see the result in the PR that way

Reply:
I see, the test result won't be reported as one of the check results.